app/javascript/components/patient/Patient.js

```diff
@@ -704,6 +705,7 @@ class Patient extends React.Component {
               </Row>
               <Row>
                 <ChangeHOH patient={this.props?.details} dependents={this.props?.dependents} authenticity_token={this.props.authenticity_token} />
+                {this.props.can_add_group && <EnrollHouseholdMember responderId={this.props.details.responder_id} />}
```

Comment:
I know that this logic/code is used twice, but was wondering if it's necessary to create an entire component just  for a button

Reply:
Seconded^

Reply:
I was borderline on this.  We have all our other household action buttons in separate components so I went with consistency

Reply:
I also did in case it is something we ever want to add anywhere else

Reply:
I'm fine with it - a button with particular functionality is a very viable & common use of a separate component.

Reply:
hmm I see, I will defer to @kylietmo on this one then